src/components/PassageDetail/PassageDetailRecord.tsx

```diff
@@ -104,6 +108,18 @@ export function PassageDetailRecord(props: IProps) {
     );
   }, [memory, passage, mediafiles]);
 
+  useEffect(() => {
+    const mediaRec = findRecord(memory, 'mediafile', mediafileId) as
+      | MediaFile
+      | undefined;
+    const performer = mediaRec?.attributes?.performedBy;
+    if (mediaRec && performer) {
```

Comment:
You don't need to check mediaRec here, just performer

Reply:
:+1